pt-BR/form_helpers.md

```diff
@@ -682,54 +682,54 @@ The form builder used also determines what happens when you do
 
 If `f` is an instance of `ActionView::Helpers::FormBuilder` then this will render the `form` partial, setting the partial's object to the form builder. If the form builder is of class `LabellingFormBuilder` then the `labelling_form` partial would be rendered instead.
 
-Understanding Parameter Naming Conventions
+Entendendo Convenções de Nomeação de Parâmetros
 ------------------------------------------
 
-Values from forms can be at the top level of the `params` hash or nested in another hash. For example, in a standard `create` action for a Person model, `params[:person]` would usually be a hash of all the attributes for the person to create. The `params` hash can also contain arrays, arrays of hashes, and so on.
+Valores de formulários podem ficar no topo do *hash* `params` ou aninhados em outro *hash*. Por exemplo, numa ação `create` padrão para um *model* *Person*, `params[:person]` normalmente seria um *hash* de todos os atributos necessários para criar *person*. O *hash* `params` também pode conter *arrays*, *arrays* de *hashes*, e por aí vai.
```

Comment:
```suggestion
Valores de formulários podem ficar no topo do *hash* `params` ou aninhados em outro *hash*. Por exemplo, numa ação `create` padrão para um *model* `Person`, `params[:person]` normalmente seria um *hash* de todos os atributos necessários para criar uma pessoa. O *hash* `params` também pode conter *arrays*, *arrays* de *hashes*, e por aí vai.
```

Fiz uma sugestão meio vaga mas essa frase tem muito itálico então tentei mudar um pouco algumas marcações o que acha?

Reply:
Segui um pouco a dinâmica lá de baixo de traduzir algumas palavras pra dar leitura também.

Reply:
Incluí as alterações em 4d5e40f4e604051e759f2247e60d028b033cc3b5